public/main.js

```diff
@@ -372,7 +373,8 @@ function renderIndThreadView(id, state){
   showView('thread');
   // Load Thread Posts
   thread[0].posts.forEach(function(post, index){  
-
+    // `post.likes` has an empty users array.
+    console.log('POST', post);
```

Comment:
When I tried to find out the `users` in `post.likes` later in the code, I found it not present. Could you investigate this?